ProcessMessage/handler.ts

```diff
@@ -309,150 +312,151 @@ export const getProcessMessageHandler = ({
   optOutEmailSwitchDate,
   isOptInEmailEnabled,
   telemetryClient
-}: IProcessMessageHandlerInput) => async (
-  context: Context,
-  input: unknown
-): Promise<ProcessMessageResult> => {
-  const createdMessageEventOrError = CreatedMessageEvent.decode(input);
-
-  if (E.isLeft(createdMessageEventOrError)) {
-    context.log.error(`${logPrefix}|Unable to parse CreatedMessageEvent`);
-    context.log.verbose(
-      `${logPrefix}|ERROR_DETAILS=${readableReport(
-        createdMessageEventOrError.left
-      )}`
-    );
-    return { kind: "FAILURE", reason: "BAD_DATA" };
-  }
+}: IProcessMessageHandlerInput): Handler =>
+  withJsonInput(async (context: Context, input: unknown): Promise<
```

Comment:
This line is the actual change: handler function has been wrapped by withJsonInput decorator